README.md

```diff
@@ -54,6 +54,50 @@ In general, liota can be installed with:
 
 It requires a Python 2.7 environment already installed.
 
+## Autostarting Liota Daemon
+For starting liotad.py in background automatically at reboot perform the following steps: 
+
+* Copy autostartliota script present in scripts folder to location:
```

Comment:
What is the location of script folder ???
And I don`t see "scripts/autostartliota" included in setup.py in this PR to exist at "/etc/liota" location.